build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+import net.minecraftforge.gradle.user.tweakers.ClientTweaker
+import net.minecraftforge.gradle.user.tweakers.TweakerExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.spongepowered.asm.gradle.plugins.MixinGradlePlugin
+
+buildscript {
+    repositories {
+        maven { setUrl("http://files.minecraftforge.net/maven") }
+        maven { setUrl("http://repo.spongepowered.org/maven") }
+    }
+
+    dependencies {
+        classpath("net.minecraftforge.gradle:ForgeGradle:${Versions.forge_gradle}")
+        classpath("org.spongepowered:mixingradle:${Versions.mixin_gradle}")
+    }
+}
+
+group = "cc.hyperium"
+version = "1.0-SNAPSHOT"
+
+plugins {
+    kotlin("jvm") version Versions.kotlin
+}
+
+apply {
+    plugin<ClientTweaker>()
+    plugin<MixinGradlePlugin>()
+}
+
+tasks.withType<KotlinCompile> {
+    kotlinOptions.jvmTarget = "1.8"
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+configure<TweakerExtension> {
+    setSuppressVersionTest(true)
+
+    version = "1.8.9"
+    mappings = "stable_22"
+
+    runDir = rootProject.file("minecraft").path
+
+    makeObfSourceJar = true
+    setTweakClass("cc.hyperium.launch.HyperiumTweaker")
+}
+
+repositories {
+    maven { setUrl("http://dl.bintray.com/kotlin/kotlin-eap") }
+    maven { setUrl("https://jitpack.io") }
+    maven {
+        name = "sponge"
```

Comment:
There is no need to add a name for this repository I believe, it would probably be cleaner if we just removed the name. In addition, the comment above applies here too.

Reply:
I can remove the name, that is just how it's done in most projects. However it would make sense to either remove the name, or name all of the repositories, for consistency.

Reply:
I think we should opt for removing the name from all, I don't think it provides any tangible benefits.